src/openrct2/platform/Platform.Linux.cpp

```diff
@@ -52,7 +52,7 @@ namespace Platform
                 if (path.empty())
                 {
                     auto home = GetFolderPath(SPECIAL_FOLDER::USER_HOME);
-                    path = Path::Combine(home, ".local/share");
+                    path = Path::Combine(home, ".config");
```

Comment:
This is not sufficient. A few lines above, it reads the environment variable `XDG_DATA_HOME`, which defaults to `.local/share`. The whole case has to be folded into `SPECIAL_FOLDER::USER_CONFIG`.

Reply:
Then how can I resolve the issue?